main.cpp

```diff
@@ -146,10 +138,6 @@ int main(int argc, char **argv) {
         return 0;
     }
 
-    initializingOnlyOnce();
-
-    game.init();
```

Comment:
The init() method of game is called a third time, yes. Hard to see if that one can go.
But if you go look game.SetupGame, then this method starts with calling init. So they're called right after eachother with nothing in between.

For the rest, I'd rather not touch it on my own without discussion. :)

Reply:
if it works properly (boots & runs a mission), then this will probably be fine. 

The real challenge is to introduce the combat state object (which I want to do after the release of 0.6.x).

Reply:
I can imagine.
I would also like to strip cGame of the Allegro and alfont initialization and move that to dedicated objects. Hide it there, so we can more easily replace Allegro.